--- /opt/code/rpmbuild/SPECS/sqlite.spec	2015-09-23 04:53:12.000000000 -0400
+++ sqlite.spec	2019-07-22 14:02:31.205271124 -0400
@@ -32,17 +32,10 @@
 
 BuildRequires: ncurses-devel readline-devel glibc-devel
 BuildRequires: autoconf
-%if %{with tcl}
 BuildRequires: /usr/bin/tclsh
 BuildRequires: tcl-devel
-%{!?fedora:%{!?tcl_version: %global tcl_version 8.6}}
-%if %{fedora} <= 20
 %{!?tcl_version: %global tcl_version 8.5}
-%else
-%{!?tcl_version: %global tcl_version 8.6}
-%endif
 %{!?tcl_sitearch: %global tcl_sitearch %{_libdir}/tcl%{tcl_version}}
-%endif
 
 %description
 SQLite is a C library that implements an SQL database engine. A large
